agilent_infiniium_scope.py

```diff
@@ -371,27 +373,26 @@ def get_trace(self, c, channel, start=1, stop=10000):
         #Transfer waveform preamble
         preamble = yield dev.query('WAV:PRE?')
         #Transfer waveform data
-        p = dev._packet(). \
-            write('WAV:DATA?'). \
-            read_raw()
+        p = dev._packet().write('WAV:DATA?').read_raw()
 
         result = yield p.send()
         binary = result['read_raw']
         binary = binary[:-1]
         #Parse waveform preamble
         preambleDict = _parsePreamble(preamble)
-        print preambleDict
+
         #Parse binary
-        trace = _parseBinaryData(binary, wordLength = wordLength)
+        trace = _parseBinaryData(binary, wordLength=wordLength)
+
         #Convert from binary to volts
-        traceVolts = ((trace*float(preambleDict['yStep'])+float(preambleDict['yOrigin'])))
+        step, origin = (float(preambleDict['yStep']),
+                        float(preambleDict['yOrigin']))
```

Comment:
If this doesn't fit on one line, I'd suggest just making two separate assignment statements:
```python
step = float(preambleDict['yStep'])
origin = float(preambleDict['yOrigin'])
```